cache.c

```diff
@@ -209,7 +209,7 @@ void cache_remove_ti(struct track_info *ti)
 static int read_cache(void)
 {
 	unsigned int size, offset = 0;
-	struct stat st = {};
+	struct stat st;
```

Comment:
Not that it matters since it works either way, but the initialization was added some months ago because the lack of an initializer also causes a warning. `{ }` is the same as `{ 0 }` but is a GNU extension.